app/serializers/initial_state_serializer.rb

```diff
@@ -109,6 +109,7 @@ def default_meta_store
       trends_as_landing_page: Setting.trends_as_landing_page,
       trends_enabled: Setting.trends,
       version: instance_presenter.version,
+      force_grouped_notifications: ENV['FORCE_GROUPED_NOTIFICATIONS'] == 'true',
```

Comment:
Should it be an env variable, or an instance-wide setting? I dont think anyone except us will enable this, as this is mainly to test performance on m.s, and having it as a setting makes it easier to enable/disable as it does not required a new deployment?